deps/nghttp2/nghttp2.gyp

```diff
@@ -12,13 +12,13 @@
       'defines': [
         'BUILDING_NGHTTP2',
         'NGHTTP2_STATICLIB',
+	'HAVE_CONFIG_H',
```

Comment:
Is the indentation right here?

Reply:
it was not, sorry, my editor apparently uses TABs for .gyp files. fixed.